be/CMakeLists.txt

```diff
@@ -239,6 +239,11 @@ install(DIRECTORY
     ${BASE_DIR}/dict/icu/uax29
     DESTINATION ${OUTPUT_DIR}/dict/icu)
 
+
+install(DIRECTORY
+    ${BASE_DIR}/dict/ik
+    DESTINATION ${OUTPUT_DIR}/dict)
```

Comment:
DESTINATION ${OUTPUT_DIR}/dict/ik

Reply:
> DESTINATION ${OUTPUT_DIR}/dict/ik

All dict files are in the `${BASE_DIR}/dict/ik` directory(`${BASE_DIR}/dict/ik/*.dict`). Setting `DESTINATION to ${OUTPUT_DIR}/dict/ik` would result in the path /dict/ik/ik/*.dict.